lib/index.js

```diff
@@ -38,6 +42,35 @@ function initialize(config) {
   });
 }
 
+function filterPreviousUploads(uploadListPath, objects) {
+  var uploadedFileList;
+  if (uploadListPath) {
+    uploadedFileList = fs.readFileSync(uploadListPath, {encoding: 'utf8'});
+    uploadedFileList = uploadedFileList.split('\n');
+    if (!uploadedFileList[uploadedFileList.length-1]) uploadedFileList.pop();
+
+    uploadedFileList = _.map(uploadedFileList, function(anUploadedFile) {
+      return JSON.parse(anUploadedFile);
+    });
+
+    uploadedFilesById = _.groupBy(uploadedFileList, 'objectId');
+  }
+
+  objects = _.filter(objects, function(aFileObject) {
+    if (!aFileObject.fileName) return false; //null fileName causes error in indexOf later
```

Comment:
In transfer.js the method `function _isParseHostedFile(fileName)` does `filename.indexOf('tfss')`